src/frontend/Sider/Behandling/Venstremeny/Historikk/Historikk.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+
+import HistorikkElement from './HistorikkElement';
+import { HistorikkHendelse } from './typer';
+import { useApp } from '../../../../context/AppContext';
+import { useBehandling } from '../../../../context/BehandlingContext';
+import DataViewer from '../../../../komponenter/DataViewer';
+import { Ressurs, byggTomRessurs } from '../../../../typer/ressurs';
+
+const Container = styled.ul`
+    margin: 0;
+    padding: 0;
+`;
+
+const Historikk: React.FC = () => {
+    const { request } = useApp();
+    const { behandling } = useBehandling();
+
+    const [historikk, settHistorikk] = useState<Ressurs<HistorikkHendelse[]>>(byggTomRessurs());
+
+    useEffect(() => {
+        request<HistorikkHendelse[], null>(`/api/sak/behandlingshistorikk/${behandling.id}`).then(
+            settHistorikk
+        );
+    }, [request, behandling.id]);
```

Comment:
Jeg tror vi burde flytte opp staten for denne til behandlingContext, sånn at vi kan trigge henting av historikken fra andre komponenter ved oppdateringer. Det kan ev. gjøres i en egen PR